vendor.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/klauspost/compress v1.15.12
 	github.com/miekg/dns v1.1.43
 	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible
-	github.com/moby/buildkit v0.10.7-0.20230206124303-b8fdb4b78da0
+	github.com/moby/buildkit v0.10.7-0.20230208155512-4f0ee09c40e2
```

Comment:
I'd like to have a tagged release for this fix if possible, but I'm not familiar with the process in BuildKit. Are we able to complete a release before this revendoring?

Reply:
We're not planning to do any actual v0.10 releases anymore, as v0.11 is already out.

Reply:
I'm really not sure how I feel about that -- this is a correctness fix for a bug in BuildKit and not a simple revendor. It really feels like there should be a tagged release, instead of us floating off the unreleased tip of the 0.10.x branch indefinitely...